solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -114,4 +114,62 @@ print(ans)
 ```
 
 </PySection>
+
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class GCDBoard {
+	static final int MAX_N = (int) 1e5 + 5;
```

Comment:
remove the +5

Reply:
I get runtime error if I don't do this

Reply:
ok so
1. remove the +5
2. replace MAX_N in gcdsuffix & prefix with MAX_N + 1

Reply:
or you can replace it with n + 1 :) not really necessary tho, but it'd be slightly faster (?) for smaller test cases